Tribler/Main/vwxGUI/channel.py

```diff
@@ -1001,29 +1001,6 @@ def startDownload(self, torrentfilename, *args, **kwargs):
             print_exc()
         return False
 
-    def startDownloads(self, filenames, *args, **kwargs):
```

Comment:
Are you sure? I think this is used while adding torrents to a channel (using the add dialog)